apps/blur/halide_blur_generator.cpp

```diff
@@ -33,7 +33,7 @@ class HalideBlur : public Halide::Generator<HalideBlur> {
 
     Func build() {
         Func blur_x("blur_x"), blur_y("blur_y");
-        Var x("x"), y("y"), xi("xi"), yi("yi");
+        Var x("x"), y("y"), xi("xi"), yi("yi"), tx("tx"), ty("ty");
```

Comment:
I think tx and ty are used for tiles in other tests. We should probably used xi, yi for the gpu thread names.